### PR TITLE
FLY-7502 showmore for cosmetics

### PR DESCRIPTION
**Pull request by BharadwajAd-Kore**
FLY-7502

### CODE REVIEW

File: src/templatemanager/templates/searchListViewTemplate/searchListViewTemplate.ts [hunk 1]
```diff
@@ -232,11 +232,8 @@ class SearchListViewTemplate {
       }
       const listHTML = $(SearchListViewTemplate.prototype.getTemplateString(result?.message[0].component.payload.template_type)).tmpl(result?.message[0].component.payload);
       $(listHTML).find(".show-more-list").remove();
-      $(
-        ".full-search-data-container [templateName=" +
-        showMoreData.templateName +
-        "]"
-      ).before($(listHTML).find(".parent-list-template").children());
+      if(result?.message[0].component.payload.template_type !== "cosmeticsTemplate") $(".full-search-data-container [templateName=" + showMoreData.templateName + "]").before($(listHTML).find(".parent-list-template").children());
```
Comment (VenkatIc-Kore) [inline src/templatemanager/templates/searchListViewTemplate/searchListViewTemplate.ts]: @BharadwajAd-Kore: This should be write it in if else condition  or else every time below condition will excuted
@SunilSi-kore

File: src/templatemanager/templates/searchListViewTemplate/searchListViewTemplate.ts [hunk 2]
```diff
@@ -540,6 +537,13 @@ class SearchListViewTemplate {
...
+                {{if isSearchSDK}}\
+                <div class="show-more-data {{if doc_count==0 || doc_count<6 || isLiveSearch || isSearch}}display-none{{/if}} show-more-list" groupName="${groupName}" templateName="${templateName}" pageNumber="${pageNumber}" fieldName="${fieldName}">\
+                <div class="searchassist-show-more-button">Show more\
+                <img src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTAiIGhlaWdodD0iNiIgdmlld0JveD0iMCAwIDEwIDYiIGZpbGw9Im5vbmUiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyI+CjxwYXRoIGZpbGwtcnVsZT0iZXZlbm9kZCIgY2xpcC1ydWxlPSJldmVub2RkIiBkPSJNNS4zNjQzNyA1LjE0OTQ5QzUuMTc0OTcgNS4zMzgzNyA0Ljg3NDgxIDUuMzQ5NDggNC42NzIzOSA1LjE4MjgyTDQuNjM1NjMgNS4xNDk0OUwwLjE1MDkyNyAwLjg3NzI2NUMtMC4wNTAzMDkxIDAuNjc2NTc5IC0wLjA1MDMwOTEgMC4zNTEyMDIgMC4xNTA5MjcgMC4xNTA1MTVDMC4zNDAzMjYgLTAuMDM4MzY2NCAwLjY0MDQ4IC0wLjA0OTQ3NzMgMC44NDI5MDkgMC4xMTcxODNMMC44Nzk2NjggMC4xNTA1MTVMNSA0LjA1OTI4TDkuMTIwMzMgMC4xNTA1MTVDOS4zMDk3MyAtMC4wMzgzNjY0IDkuNjA5ODggLTAuMDQ5NDc3MyA5LjgxMjMxIDAuMTE3MTgzTDkuODQ5MDcgMC4xNTA1MTVDMTAuMDM4NSAwLjMzOTM5NyAxMC4wNDk2IDAuNjM4NzMxIDkuODgyNSAwLjg0MDYwN0w5Ljg0OTA3IDAuODc3MjY1TDUuMzY0MzcgNS4xNDk0OVoiIGZpbGw9IiM1RjYzNjgiLz4KPC9zdmc+Cg==">\
```
Comment (VenkatIc-Kore) [inline src/templatemanager/templates/searchListViewTemplate/searchListViewTemplate.ts:548]: @BharadwajAd-Kore : Don't use base 64 images
@SujithPu-Kore

**Review by VenkatIc-Kore — COMMENTED** [review]
could you please fix the commented lines

File: src/templatemanager/templates/searchListViewTemplate/searchListViewTemplate.ts (hunk 2 above)
Comment (VenkatIc-Kore) [reply src/templatemanager/templates/searchListViewTemplate/searchListViewTemplate.ts:548]: @BharadwajAd-Kore : show more i18n change missed, after this approval please add that change
@SunilSi-kore

**Review by VenkatIc-Kore — APPROVED** [review]
show more button i18n change need to add . remaining looks good

**Review by SunilSi-kore — APPROVED** [review]
Approved

**Pull request merged**